### PR TITLE
Add Python 3.6 and drop Python 3.3

### PR DESCRIPTION
**Pull request by jamadden**
- Badges
- Enable coverage and coveralls
  - 100% coverage
- DRY with dependencies.
- Test compatibility with new zope.container that fixes
  OrderedContainer to store unicode keys.
  - Convert failing doctest into real test.

### CODE REVIEW

File: src/zope/copypastemove/tests/test_clipboard.py [hunk 1]
```diff
@@ -82,10 +82,8 @@ def testClearContents(self):
...
-    return unittest.TestSuite((
-        unittest.makeSuite(PrincipalClipboardTest),
-        ))
+    return unittest.defaultTestLoader.loadTestsFromName(__name__)
 
-if __name__=='__main__':
-    unittest.main(defaultTest='test_suite')
 
+if __name__ == '__main__':
+    unittest.main(defaultTest='test_suite')
```
Comment (icemac) [inline src/zope/copypastemove/tests/test_clipboard.py]: Is this really still needed?

Comment (jamadden) [inline src/zope/copypastemove/tests/test_clipboard.py]: Almost certainly not 😄 

OTOH, it was there, and aside from some bad whitespace, it didn't seem to be hurting much.

I admit, I am totally inconsistent about the way I clean those up. I think I usually try to just leave them and fix the whitespace, but I have also removed some by request or my own whims...

**Comment by jamadden** [comment]
Thanks! Can we please get a PyPI release when convenient (or access)?

**Comment by icemac** [comment]
@jamadden `jamadden` is now owner of `zope.copypastemove` on PyPI.

**Comment by jamadden** [comment]
Thanks! 4.1.0 has been released to PyPI.

**Pull request merged**